docs/docusaurus/docs/core/connect_to_data/configure_credentials/_access_key_pair.md

```diff
@@ -0,0 +1,24 @@
+
+To use key-pair authentication for Snowflake, you will pass the private key as a connection argument with `kwargs` in addition to passing connection details with the `connection_string` parameter. Here's an example of how to access your private key in Python.
+
+```python title="Python"
+import pathlib
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization
```

Comment:
Note for reviewer: The usage example in the [implementation PR](https://github.com/great-expectations/great_expectations/pull/10031) had a couple more imports here, but they didn't appear to be used so I removed them

```
from cryptography.hazmat.primitives.asymmetric import rsa
from cryptography.hazmat.primitives.asymmetric import dsa
```

let me know if that was a bad assumption and I need to restore these

Reply:
Removing these look fine to me.